src/test/java/org/apache/commons/collections4/CollectionUtilsTest.java

```diff
@@ -687,7 +687,9 @@ public void get() {
         assertEquals(2, CollectionUtils.get((Object) collectionA, 2));
         assertEquals(2, CollectionUtils.get((Object) collectionA.iterator(), 2));
         final Map<Integer, Integer> map = CollectionUtils.getCardinalityMap(collectionA);
-        assertEquals(map.entrySet().iterator().next(), CollectionUtils.get((Object) map, 0));
+        Map<Integer, Integer> linkedMap = new LinkedHashMap<>();
```

Comment:
Add a comment to explain why this is done:
```Java
// Test assumes a defined iteration order so convert to a LinkedHashMap
```
Please use final:
```Java
final Map<Integer, Integer> linkedMap = ...
```

Reply:
Thank you so much for the clarification. I have updated the code accordingly.